src/components/calendar-toolbar.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { Button, Container, Padding, Tooltip } from '@zextras/carbonio-design-system';
+
+export const CalendarToolbar = ({
+	dateLabel,
+	resetButtonLabel,
+	rightArrowLabel,
+	leftArrowLabel,
+	onTodayAction,
+	onRightArrowAction,
+	onLeftArrowAction
+}: {
+	dateLabel: string;
+	resetButtonLabel: string;
+	rightArrowLabel: string;
+	leftArrowLabel: string;
+	onTodayAction: () => void;
```

Comment:
I suggest renaming the prop to onResetAction for both consistency and semantics.